### DISCUSSION TITLE
希望支持K8s或者docker的部署方案

### DISCUSSION THREAD

**Discussion by slarkzhu** [Tencent/bk-ci#6777] [General]
目前基于开源版源码部署较为复杂，基于社区版部署成本又比较高，docker版本也只是个demo。希望支持正式的k8s或者docker的部署方案

**Comment by fayewong-v5** [comment]
蓝鲸7.0已经开始支持容器化部署：https://bk.tencent.com/docs/document/7.0/172/14038
